src/GUI/EFCorePowerTools/Dialogs/PickTablesDialog.xaml

```diff
@@ -53,11 +53,11 @@
                         <Setter Property="Margin" Value="5, 0, 0, 0"/>
                     </Style>
                     <Style x:Key="TableNameTextBlockStyle" TargetType="TextBlock">
-                        <Style.Triggers>
-                            <DataTrigger Binding="{Binding Model.HasPrimaryKey}" Value="False">
+                        <!--<Style.Triggers>
```

Comment:
Can we just remove this?

Reply:
I left it there to discuss with you whether this feature is still needed or not. The HasPrimaryKey was always set to true via code which made impossible to have the check disabled. If this feature is no longer needed I can remove it.

Reply:
You can remove it